eng/pipelines/templates/steps/workload-publish.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  publishToFeed: false
+  feedForPublishing: ''
+
+steps:
+- task: 1ES.PublishNuget@1
+  condition: ${{ eq(parameters.publishToFeed, true) }}
+  displayName: 🟣 Publish NuGet package
```

Comment:
Contextually, when we add publishing to NuGet.org, this would probably be something like `Publish package to AzDO` or something like that.